packages/venia-concept/src/components/Checkout/paymentsForm.js

```diff
@@ -1,59 +1,45 @@
 import React, { Component } from 'react';
 import { Form } from 'informed';
-import { array, bool, func, shape, string } from 'prop-types';
+import { bool, func, shape, string } from 'prop-types';
 
+import BraintreeDropin from './braintreeDropin';
 import Button from 'src/components/Button';
-import Label from './label';
-import Select from 'src/components/Select';
 
 import classify from 'src/classify';
 import defaultClasses from './paymentsForm.css';
 
 class PaymentsForm extends Component {
     static propTypes = {
-        availablePaymentMethods: array.isRequired,
         cancel: func.isRequired,
         classes: shape({
             body: string,
             footer: string,
             heading: string,
             paymentMethod: string
         }),
-        paymentMethod: string,
         submit: func.isRequired,
         submitting: bool
     };
 
-    static defaultProps = {
-        availablePaymentMethods: [{}]
+    // TODO: don't actually keep the state here, move to Redux
+    state = {
+        isRequestingPaymentNonce: false
     };
 
     render() {
-        const {
-            availablePaymentMethods,
```

Comment:
I realize now that support for multiple payment methods is being removed in this PR so some of my other concerns around the API request for payment methods is moot.

I really don't have enough context on the Venia project yet to know, but maybe @ericerway or @awilcoxa could let us know whether or not we should keep this around?